scripts/linters/other_files_linter.py

```diff
@@ -237,6 +241,37 @@ def check_webpack_config_file(self):
         return concurrent_task_utils.TaskResult(
             name, failed, error_messages, error_messages)
 
+    def check_strict_ts_config(self):
+        """Checks if the files in strict TS config are sorted
+        alphabetically.
+
+        Returns:
+            TaskResult. A TaskResult object representing the result of the lint
+            check.
+        """
+        name = 'Sorted strict TS config'
+
+        failed = False
+        error_messages = []
+
+        strict_ts_config = json.load(
+            python_utils.open_file(STRICT_TS_CONFIG_FILEPATH, 'r'))
+
+        # Remove .ts extension from filepath for sorting to ensure that
+        # spec files are always below the main files.
+        files = [path[:-3] for path in strict_ts_config['files']]
+        sorted_files = sorted(files)
+
+        if files != sorted_files:
+            failed = True
+            error_message = (
+                'Files in %s are not alphabetically sorted.' % (
```

Comment:
As this list is going to be very big should we inform the initial line number where the item is not sorted? (optional)

Reply:
The dev would know the files they added are a problem, so I think it won't be difficult to spot out the mistake.